third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2939,7 +2939,6 @@ func expandDatabaseEncryption(configured interface{}) *containerBeta.DatabaseEnc
 	}
 }
 
-<% unless version == 'ga' -%>
 func expandReleaseChannel(configured interface{}) *containerBeta.ReleaseChannel {
```

Comment:
I think you need a nil check here before the attempt at a cast; if l == nil { return nil }.